docs/data/material/components/tabs/BasicTabs.js

```diff
@@ -18,7 +18,7 @@ function CustomTabPanel(props) {
     >
       {value === index && (
         <Box sx={{ p: 3 }}>
-          <Typography>{children}</Typography>
+          <Typography component={`span`}>{children}</Typography>
```

Comment:
```suggestion
          <Typography component="span">{children}</Typography>
```

I guess you don't need the curly braces and backticks here :)

Reply:
you're right @danilo-leal, thanks :)